apps/challenges/models.py

```diff
@@ -94,6 +94,7 @@ def is_active(self):
 
```

Comment:
Please add `phase_id` and `phase_split_id` in the in the `apps/challenges/admin.py` in `list_display` field.

Reply:
Okay!